devstats/__main__.py

```diff
@@ -46,6 +47,8 @@ def query(repo_owner, repo_name, outdir):
         print("You need to set GRAPH_API_KEY")
         sys.exit()
 
+    start_date = "2020-01-01T00:00:00Z"
```

Comment:
Is it possible to pass this in via the cli?

Reply:
Sure I look at adding that

Reply:
done in latest commit